crates/turborepo-cache/src/lib.rs

```diff
@@ -80,3 +86,18 @@ pub struct CacheResponse {
     source: CacheSource,
     time_saved: u32,
 }
+
+#[derive(Debug, Default)]
+pub struct CacheOpts<'a> {
```

Comment:
Moved these structs here to avoid a circular dependency between `turborepo-lib` and `turborepo-cache`